File: app/src/main/java/com/ianarbuckle/dublinbushelper/helper/LocationHelper.java
package com.ianarbuckle.dublinbushelper.helper;

import android.support.v4.app.Fragment;

import com.google.android.gms.maps.GoogleMap;

/**
 * Created by Ian Arbuckle on 19/02/2017.
 *
 */

public interface LocationHelper {
  boolean checkLocationPermission(Fragment fragment);
  void initMap(GoogleMap googleMap);
  void onRequestPermission();
}

File: app/src/main/java/com/ianarbuckle/dublinbushelper/helper/LocationHelper.kt
package com.ianarbuckle.dublinbushelper.helper

import android.support.v4.app.Fragment

import com.google.android.gms.maps.GoogleMap

/**
 * Created by Ian Arbuckle on 19/02/2017.

 */

interface LocationHelper {
    fun checkLocationPermission(fragment: Fragment): Boolean
    fun initMap(googleMap: GoogleMap)
    fun onRequestPermission()
}
